### DISCUSSION TITLE
[API Design] Download component without getLink method

### DISCUSSION THREAD

**Discussion by stefanseifert** [adobe/aem-core-wcm-components#1859] [Ideas]
the download component model seems to be the only model that was not switched to the new Link concept. is this component obsolete, or is it an oversight?

---

this discussion is taken over from the [mailing list](https://groups.google.com/d/msgid/aem-core-components-dev/AS8PR07MB7238137D48504B293171B7F1B8C89%40AS8PR07MB7238.eurprd07.prod.outlook.com) where it did not gather any attention (except from @ky940819).

**Comment by jckautzmann** [comment]
Introducing a `Link getLink()` to the Download component model would be a breaking change. We will introduce this change with the future v2 version.
